lib/writeGltf.js

```diff
@@ -19,31 +28,24 @@ function writeGltf(gltf, outputPath, embed, embedImage, createDirectory, callbac
     if (outputExtension !== '.gltf') {
         throw new DeveloperError('Invalid output path extension.');
     }
+
     // Create the output directory if specified
     if (createDirectory) {
         outputPath = path.join(path.dirname(outputPath), 'output', path.basename(outputPath));
+        fsExtra.ensureDirSync(path.dirname(outputPath));

```

Comment:
You don't need this because `outputJson` will create the dir if it doesn't exist.